app/views/errors.py

```diff
@@ -89,7 +89,9 @@ def _render_error_page(status_code):
         flask_render_template(
             'errors/error.html',
             status_code=status_code,
-            analytics_ua_id=current_app.config['EQ_UA_ID'],
+            gtm_id=current_app.config['EQ_GTM_ID'],
+            gtm_auth=current_app.config['EQ_GTM_AUTH'],
+            gtm_preview=current_app.config['EQ_GTM_PREVIEW'],
```

Comment:
These three lines are repeated 8 times. I think we should do something about it. There is a `render_template` method in `template_helper.py` that could help.

Reply:
Made some changes to how render_template is used throughout the app to reduce this duplication.